GOVERNANCE.md

```diff
@@ -4,51 +4,163 @@ This document defines governance policies for the Kiali project.
+Maintainer.
+
+Testers do not need to be Maintainers. Also, Maintainers do not need to be Testers. Yet,
+both roles aren't mutually exclusive.
+
+Current list of testers (alphabetically ordered):
+
+* [FilipB](https://github.com/FilipB)
+* [mattmahoneyrh](https://github.com/mattmahoneyrh)
+* [pbajjuri20](https://github.com/pbajjuri20)
+* [prachiyadav](https://github.com/prachiyadav)
+* [skondkar](https://github.com/skondkar)
+
+### Becoming a Tester
+
+To become a Tester you need to:
+
+* actively participate in testing code changes of the Kiali project
+  * testing pull requests for 3 months or more,
+  * find 5 non trivial defects in Kiali,
+  * occassionally, do testing over `master` branches to find broken features
+* ability to collaborate with the team,
+* ability to document testing procedures of Kiali features and to update existing documentation if features change,
+* understanding of team policies and processes.
+
+You can propose yourself to apply for the Tester role, or a Maintainer can do it for you.
+Proposals must be posted by opening a pull request
+modifying the testers list to add the new proposed tester, or via
+a [GitHub discussion](https://github.com/kiali/kiali/discussions/new) under the
+Governance category. The following information must be provided:
+
+* nominee's GitHub username
+* a list of pull requests (top 5) tested by the nominee.
+* a list of documented test cases (if any)
+
+Formalization of the Tester role is granted if a simple majority vote
+among current maintainers succeeds.
+
+## Leaders
+
+Kiali leaders are Maintainers who have a broad knowledge about the Kiali project, its goals and vision.
+They may also be aware on how the ecosystem related to Kiali is evolving, and may also be aware of related projects.
+Thus, they are able to guide and mentor other Maintainers, give direction, and set priorities to the Kiali project.
+
+Current list of leaders (alphabetically ordered):
+
+* [abonas](https://github.com/abonas)
+* [jshaughn](https://github.com/jshaughn)
+* [lucasponce](https://github.com/lucasponce)
+
+### Becoming a Leader
+
+To become a Leader, you must first be a Maintainer. Then, a new Leader must be proposed
+by an existing Maintainer by opening a pull request modifying the leaders list to add the new
+proposed leader, or via a by opening a [GitHub discussion](https://github.com/kiali/kiali/discussions/new)
+under the Governance category. The following information must be provided:
+
+* nominee's GitHub username,
+* an explanation of why the nominee should be a leader.
+
+Leadership is granted if a simple majority vote among current maintainers succeeds.
+
+## Inactivity
+
+It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+* Inactivity is measured by:
```

Comment:
contribution AND communication or one of them ?

Reply:
I think it is contribution _and_ communication... 

It really depends on what is the scope of "contribution", because I understand that (for us) even ideas dropped as issue comments can be seen as contributions. If this is true, I guess that anyway you need to be totally absent to fall in inactivity.